IGD device only has last 1 page used by GTT. this should align to AGP gart code.

Signed-off-by: Shaohua Li <shaohua.li@intel.com>
---
 drivers/gpu/drm/i915/i915_dma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux/drivers/gpu/drm/i915/i915_dma.c
===================================================================
--- linux.orig/drivers/gpu/drm/i915/i915_dma.c	2009-03-13 15:36:12.000000000 +0800
+++ linux/drivers/gpu/drm/i915/i915_dma.c	2009-03-13 15:37:26.000000000 +0800
@@ -880,7 +880,7 @@ static int i915_probe_agp(struct drm_dev
 	 * Some of the preallocated space is taken by the GTT
 	 * and popup.  GTT is 1K per MB of aperture size, and popup is 4K.
 	 */
-	if (IS_G4X(dev))
+	if (IS_G4X(dev) || IS_IGD(dev))
 		overhead = 4096;
 	else
 		overhead = (*aperture_size / 1024) + 4096;

